sqlglot/parser.py

```diff
@@ -2223,6 +2223,8 @@ def _parse_query_modifiers(
                     key, expression = parser(self)
 
                     if expression:
+                        if this.args.get(key):
```

Comment:
I am also not sure if `this.args` is supposed to be an internal variable to the `Expression` class or not, but I am not able to find any public methods for it